internal/collector/telemetry/process_telemetry.go

```diff
@@ -20,17 +20,30 @@ import (
 	"runtime"
 	"time"
 
-	"github.com/prometheus/procfs"
+	"github.com/shirou/gopsutil/process"
 	"go.opencensus.io/stats"
 	"go.opencensus.io/stats/view"
 )
 
 // ProcessMetricsViews is a struct that contains views related to process metrics (cpu, mem, etc)
 type ProcessMetricsViews struct {
+	prevTimeUnixNano int64
 	ballastSizeBytes uint64
 	views            []*view.View
 	done             chan struct{}
-	proc             *procfs.Proc
+	proc             *process.Process
+}
+
+var mUptime = stats.Float64(
+	"process/uptime",
+	"Uptime of the process",
+	stats.UnitSeconds)
+var viewProcessUptime = &view.View{
+	Name:        mUptime.Name(),
+	Description: mUptime.Description(),
+	Measure:     mUptime,
+	Aggregation: view.Sum(),
```

Comment:
Why not last value? Then you wouldn't need to calculate delta (i.e. record `prevTimeUnixNano`) and just report the current value, as far as I undertand.

Reply:
Tbh I'm not super familiar with the OpenCensus APIs, and this may not be the right way to do this. But the reason for choosing `Sum` was so this is created as a `Counter` metric. If I use `LastValue`, this is created as a `Gauge`.